.github/workflows/build.yaml

```diff
@@ -39,12 +40,14 @@ jobs:
         uses: actions/checkout@v2
       - name: Install dependencies
         run: cd frontend && yarn install
-      - name: Run Prettier
-        run: cd frontend && npm run prettify
-        continue-on-error: true
       - name: Run ESLint
         run: cd frontend && npm run lint
-        continue-on-error: true
+      - name: TSC
+        run: yarn tsc
+        working-directory: ./frontend
+      - name: Build - Webpack
+        run: yarn run build
+        working-directory: ./frontend
```

Comment:
`make build-frontend-amd64`  runs the `yarn build` so it is duplicated.

Do checkout the commands the Dockerfile below to verify the same:
https://github.com/SigNoz/signoz/blob/develop/frontend/Dockerfile

Reply:
aah my bad ... please re-review the code one more time i have removed the duplicate steps